### ISSUE TITLE
Meeting Type Descriptions??

### ISSUE THREAD

**Issue by enderv** [CAWSCIT/ca-meeting-finder-issue-tracker#23]
**Is your feature request related to a problem? Please describe.**
Should we have Descriptions for open/closed or any other type?? 

Default open/close has a description based on AA descriptions and was removed to address #22 but do we want to replace it?

**Comment by TheSoberJewel** [comment]
I realize I am not part of the Development Team, but my input is YES, keep the descriptions.  Just update them to be specific to Cocaine Anonymous.  Think of the newcomer. Not everyone is going to know what an 'open' or 'closed' meeting is.  I would actually suggest expanding this concept to provide definitions for the 5-10 most common meeting types we have in C.A.  Again, think of the newcomer who might be looking at this app with little to no idea of what 12 Step Recovery is or how it functions.

**Comment by KalobTaulien** [comment]
@enderv you've unset a bunch of descriptions in https://github.com/CAWSCIT/ca-mobile-app/pull/51/files 

I guess the questions now are (in regards to C.A.):
* What do we want to define `O` as (Open)?
* What do we want to define `C` as (Closed)?

**Comment by enderv** [comment]
Yeah we would need a definition and translations from world? to add these back in. I would say we don't add them as every other meeting type does not have a description.

**Comment by KalobTaulien** [comment]
Related to #22
